packages/terminal/src/node/shell-process.ts

```diff
@@ -41,7 +41,7 @@ export class ShellProcess extends TerminalProcess {
     constructor(
         @inject(ShellProcessOptions) options: ShellProcessOptions,
         @inject(ProcessManager) processManager: ProcessManager,
-        @inject(ILogger) logger: ILogger
+        @inject(ILogger) @named("package") logger: ILogger
```

Comment:
I think it should be a constant

just curious, why `package`?

Reply:
that was wrong... updated with the pkg name